features/product/viewing_products/seeing_products_lowest_price_before_the_discount.feature

```diff
@@ -90,7 +90,7 @@ Feature: Seeing the product's lowest price before the discount
         When I check this product's details
         Then I should see "$20.00" as its lowest price before the discount
 
-    @todo
+    @ui
     Scenario: Seeing the lowest price information on a product with same discount repeated
         Given the store has a product "Wyborowa Vodka" priced at "$37.00"
         And this product's price changed to "$20.00" and original price changed to "$37.00"
```

Comment:
Why the last one has been left with @todo? Is it intentionally?

Reply:
It will be covered in another task - checking if the channel has enabled displaying that price